react-components/src/components/RevealTopbar/RuleBasedOutputsContainer.tsx

```diff
@@ -0,0 +1,103 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { useState, type ReactElement, useEffect, useMemo } from 'react';
+
+import { Button, Dropdown, Flex, Icon, Menu, Switch } from '@cognite/cogs.js';
+
+import { RuleBasedOutputs } from '../RuleBasedOutputs/RuleBasedOutputs';
+import { RULE_BASED_OUTPUTS_VIEW } from '../RuleBasedOutputs/constants';
+import { FdmSDK } from '../../utilities/FdmSDK';
+import { useSDK } from '../RevealCanvas/SDKProvider';
+import { type RuleAndEnabled } from '../RuleBasedOutputs/types';
+import { useTranslation } from '../i18n/I18n';
+
+export const RuleBasedOutputsContainer = (): ReactElement => {
+  const [ruleInstances, setRuleInstances] = useState<RuleAndEnabled[]>();
+  const [currentRuleSetEnabled, setCurrentRuleSetEnabled] = useState<any>();
+  const sdk = useSDK();
+  const fdmSdk = useMemo(() => new FdmSDK(sdk), [sdk]);
+
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+    const getAllInstances = async () => {
+      const versionedPropertiesKey = `${RULE_BASED_OUTPUTS_VIEW.externalId}/${RULE_BASED_OUTPUTS_VIEW.version}`;
+
+      const filter = {
+        in: {
+          property: [RULE_BASED_OUTPUTS_VIEW.space, versionedPropertiesKey, 'shamefulOutputTypes'],
+          values: ['color']
+        }
+      };
+      const result = await fdmSdk.filterAllInstances(filter, 'node', RULE_BASED_OUTPUTS_VIEW);
+
+      const rulesAndEnabled: RuleAndEnabled[] = [];
+      result.instances.forEach((instance) => {
+        rulesAndEnabled.push({
+          isEnabled: false,
+          rule: instance
+        });
+      });
+
+      setRuleInstances(rulesAndEnabled);
+    };
+
+    void getAllInstances();
+  }, []);
+
+  const onChange = (data: { target: { id: any; checked: boolean } }): void => {
+    ruleInstances?.forEach((item) => {
+      item.isEnabled = false;
+    });
+    const selectedRule = ruleInstances?.find((item) => {
+      return item.rule.properties.id === data.target.id && data.target.checked;
+    });
+
+    setRuleInstances(ruleInstances);
+    setCurrentRuleSetEnabled(selectedRule);
+
+    if (selectedRule === undefined) return;
+    selectedRule.isEnabled = data.target.checked;
+  };
+
+  return (
+    <>
+      <Dropdown
```

Comment:
Does this dropdown have text in it? Or is it just an icon? Since we for now will only be targetting the vertical toolbar, we will need a version with only the icon

Reply:
Updated with just the icon here:
https://github.com/cognitedata/reveal/pull/4210/commits/e4c1637a2c6f4412f8fa79861d3d37844a646860